src/Drush/Command/BltDoctorCommand.php

```diff
@@ -199,7 +200,7 @@ protected function setStacksEnabled() {
     }
 
     if (file_exists($this->repoRoot . '/Vagrantfile')
-      && file_exists($this->repoRoot . '/project.local.yml')
+      && file_exists($this->repoRoot . '/blt/project.local.yml')
```

Comment:
I think this was a bug? Fixing the place it's looking for the `project.local.yml` file.